src/pages/home/_markets_fold.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { ReactElement, useState } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { Flex, SectionContainer } from 'components/containers'
+import { Localize } from 'components/localization'
+import { Carousel, Header, QueryImage, Text } from 'components/elements'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+import device from 'themes/device.js'
+
+const FoldWrapper = styled(SectionContainer)`
+    max-width: 100%;
+    padding: 120px 20px;
+
+    @media ${device.tablet} {
+        padding: 40px 16px 12px;
+    }
+`
+
+const FoldContainer = styled(Flex)`
+    flex-direction: column;
+    margin: 0 auto;
+`
+
+const ItemWrapper = styled.div`
+    z-index: 4;
+    height: 320px;
+`
+
+const CarouselItemContainer = styled(Flex)`
```

Comment:
```suggestion
const CarouselItemContainer = styled.div`
```

Reply:
This change seems to break the component. But I can just passed the `flex-direction`, `jc`,  and `width` as props to the Flex container. I hope that should do